recipes/esys-escript/meta.yaml

```diff
@@ -0,0 +1,73 @@
+
+package:
+    name: esys-escript
+    version: 5.4
+source:
+    - svn_url: https://svn.geocomp.uq.edu.au/svn/esys13/trunk
+      svn_rev: 6850
+      folder: escript
+build:
+    skip: True  # [osx]
+    skip: True  # [win]
+    number: '0'
+    rpaths:
+        - ${PREFIX}/bin
+        - ${PREFIX}/include
+        - ${PREFIX}/lib
+        - ${SP_DIR}
+requirements:
+    build:
+        - {{ compiler('cxx') }}
+        - svn
+        - scons 3.0.5 py27_0
+        - cmake
+        - make
+        - openmp
+    host:
+        - boost-cpp 1.70
+        - boost
+        - python 2.7.15 
+        - suitesparse 4.5.6 
+        - matplotlib 2.2.4 
+        - gdal 2.4.1 
+        - pyproj 2.1.3 
+        - scipy 1.2.1 
+        - openmp
+        - sympy
+    run:
+        - boost-cpp 1.70.0.*
+        - boost
+        - numpy >=1.14.6,<2.0a0
+        - python >=2.7,<2.8.0a0
+        - suitesparse 4.5.6 
+        - matplotlib 2.2.4 
+        - gdal 2.4.1 
+        - pyproj 2.1.3 
+        - scipy 1.2.1 
```

Comment:
Do you really need such strong pins?

Reply:
I probably do not. I require fairly recent versions of boost and scons, as well as python 2.7, but otherwise I do not require strong pins. I have updated the meta.yaml file accordingly.